monai/bundle/scripts.py

```diff
@@ -748,35 +787,17 @@ def run_workflow(workflow: str | BundleWorkflow | None = None, args_file: str |
         python -m monai.bundle run_workflow --meta_file <meta path> --config_file <config path>
 
         # Set the workflow to other customized BundleWorkflow subclass:
-        python -m monai.bundle run_workflow --workflow CustomizedWorkflow ...
+        python -m monai.bundle run_workflow --workflow_name CustomizedWorkflow ...
 
     Args:
-        workflow: specified bundle workflow name, should be a string or class, default to "ConfigWorkflow".
+        workflow_name: specified bundle workflow name, should be a string or class, default to "ConfigWorkflow".
```

Comment:
This argument's name was changed because it was duplicated here and may cause confusion.
https://github.com/Project-MONAI/MONAI/blob/2229c4d9cb02b093054a04ede442953497df673e/monai/bundle/workflows.py#L42

cc @Nic-Ma

Reply:
Please don't change it, it's aligned with:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/workflows.py#L41

Thanks.

Reply:
Hi @Nic-Ma,  the one I changed is not the `workflow` in BundleWorkflow, it's the workflow name you send to the `run_workflow`.
https://github.com/Project-MONAI/MONAI/blob/2daabf917a1e6afbc738d22db602f4859672a90d/monai/bundle/scripts.py#L754

Do you still think the change does not make sense?

Reply:
we may change all the `workflow: specifies the workflow type` arg to `workflow_type`, `workflow` as an input argument name seems a bit vague..

Reply:
Hi @wyli , OK, looks reasonable to me.

Thanks.

Reply:
Hi @wyli and @Nic-Ma, I have changed `worklfow` to `workflow_type` in this [commit](https://github.com/Project-MONAI/MONAI/pull/6835/commits/0039860de8b02e0e62f6aa935d7bdb88dd5c545e), could you please help check? Thanks!